artfiles.py

```diff
@@ -5,13 +5,14 @@
 # Copyright (c) 2016, InnoGames GmbH
 #
 
+from argparse import ArgumentParser
 from time import time
 from urllib import urlopen
-from argparse import ArgumentParser
 
 
 def parse_args():
     parser = ArgumentParser()
+    parser.add_argument('--prefix', default='artfiles')
```

Comment:
It is not.  We will supply the existing one as an argument.  I wanted the scripts to have more consistent defaults.